include/nbl/builtin/hlsl/glsl_compat/core.hlsl

```diff
@@ -182,21 +182,23 @@ struct bitfieldExtract<T, isSigned, false>
     }
 };
 
+// TODO: redundant T
 template<typename T>
 struct bitfieldExtract<T, true, true>
 {
     static T __call( T val, uint32_t offsetBits, uint32_t numBits )
     {
-        return spirv::bitFieldSExtract<T>( val, offsetBits, numBits );
+        return spirv::bitFieldExtract( val, offsetBits, numBits );
     }
 };
 
+// TODO: redundant T
 template<typename T>
 struct bitfieldExtract<T, false, true>
 {
     static T __call( T val, uint32_t offsetBits, uint32_t numBits )
     {
-        return spirv::bitFieldUExtract<T>( val, offsetBits, numBits );
+        return spirv::bitFieldExtract( val, offsetBits, numBits );
     } 
 };
```

Comment:
Inline SPIR-V should have exact same names as `Op` enums, except without `Op` and the leading letter should be lowercase

Reply:
alos why do you think `T` is redundant?

P.S. now I realize the overload choice chould have been done nicer with `enable_if_t`